doc/adr/0002-summary-of-decisions-thus-far.md

```diff
@@ -0,0 +1,85 @@
+# 2. Summary of decisions thus far
+
+Date: 2024-10-09
+
+## Status
+
+Accepted
+
+## Context
+
+This is a high-level summary of previous decisions that have been made on this
+project. ADRs will be used for all new architectural decisions.
+
+## Decision
+
+Below is a list with the specific explanations on the architectural decisions
+that have been made thus far in this project.
+
+#### Use Django and Django Admin.
+
+Previously, a Web 1.0 Python application was being built to create this
+microservice. It was felt that utilizing Django and the built in Admin service
```

Comment:
Old:
> It was felt that utilizing Django and the built in Admin service
could easily support the projected user base size of the Affiliations
Microservice and it would increase the speed of delivery on this project.

New:
> It was felt that utilizing Django's built-in admin site
could easily support the projected user base size of the Affiliations
Microservice and it would increase the speed of delivery on this project.

Reply:
Made this change!